test/rekt/broker_test.go

```diff
@@ -309,7 +309,7 @@ func TestBrokerSupportsAuthZ(t *testing.T) {
 	ctx, env := global.Environment(
 		knative.WithKnativeNamespace(system.Namespace()),
 		knative.WithLoggingConfig,
-		knative.WithTracingConfig,
+		knative.WithObservabilityConfig,
```

Comment:
Not for this PR, but consider moving the `global.Environment` setup to a helper function, since it seems to be repeated a lot with the same arguments.  In particular, it makes it hard to figure out where the values are _supposed_ to be different, because the eye skips over things as boilerplate.